index.html

```diff
@@ -0,0 +1,150 @@
+        <img src="sources/images/twitter-white-logo.png" alt="twitter">
+        <h4>English</h4>
+        <h4>My Page</h4>
+        <h4>Logout</h4>
+    </div>
+    <header class="header">
+        <div id="stickers">
+            <img src="sources/mobile-version/images/cool-face.png" alt="emoji" class="emoji1">
+            <img src="sources/mobile-version/images/drink logo.png" alt="drink" class="emoji2">
+            <img src="sources/mobile-version/images/camera-logo.png" alt="camera" class="emoji3">
+            <img src="sources/mobile-version/images/crazy-logo.png" alt="crazy" class="emoji4">
+            <img src="sources/mobile-version/images/party-logo.png" alt="party" class="emoji5">
+            <img src="sources/mobile-version/images/cool-face.png" alt="emoji" class="emoji6">
+            <img src="sources/mobile-version/images/drink logo.png" alt="drink" class="emoji7">
+            <img src="sources/mobile-version/images/camera-logo.png" alt="camera" class="emoji8">
+            <img src="sources/mobile-version/images/fast-forward-logo.png" alt="fast" class="emoji9">
+            <img src="sources/mobile-version/images/shuffle-logo.png" alt="shuffle" class="emoji10">
+        </div>
+        <div class="background-image">
+            <div class="overlay"></div>
+            <ul class="nav-list">
+                <img src="sources/mobile-version/images/tiny-logo.png" alt="" class="close-hamburger">
+                <a href="index.html"><img src="sources/mobile-version/images/logo.png" alt="" class="logo-desktop"></a>
+                <div class="list-container">
+                    <li><a href="about.html" class="about">About</a></li>
+                    <li><a href="" class="menu-links">Program</a></li>
+                    <li><a href="" class="menu-links">Join</a></li>
+                    <li><a href="" class="menu-links">Sponsors</a></li>
+                    <li class="menu-links-project"><a href="" class="menu-links">Project X</a></li>
+                </div>
+            </ul>
+            <nav class="navbar">
+                <img src="sources/mobile-version/images/tiny-logo.png" alt="" class="logo">
+                <img src="sources/mobile-version/images/hamburger-button.png" alt="" class="hamburger">
+            </nav>
+            <h2 class="header-h2">"Hello! Project X"</h2>
+            <h1 class="header-h1">PROJECT X. <br> A NEW EXPERIENCE IN <br> 2023</h1>
+            <div class="container-p-desktop">
+                <p class="header-p-desktop">If you are searching for a place to pass your vacations, and have curiosity whats happening in the party. This is your place, our main objective and principal because this project started is keep the young spirit, and spread the happines of the tecno music around the world, for ONLY ONE WEEK, what are you waiting for to come to the party!?</p>
+            </div>
+            <h2 class="header-day-desktop">2023.7.7(SUN) ~ 13(SAT)</h2>
```

Comment:
- [ ] as mentioned by the previous reviewer, kindly use the correct color (#272A31) for this date section, like in the original design.
yours
![Screenshot (1170)_LI](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/4d31a353-1d04-439c-bc72-195bf227d4da)
original
![Screenshot (1164)](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/517e9543-48ac-4842-ac54-3dc045cf8abd)